### PR TITLE
chore: refactor load vault + add harvests example

### PR DESCRIPTION
**Pull request by axejintao**
# Summary

- update `loadProductionVaults` for increased user friendliness
- added examples folder with loading vaults + harvests listing examples
- updated vaults v1.5 latest abi

### CODE REVIEW

**Review by NamelsKing — APPROVED** [review]

File: examples/harvests.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  // construct the sdk for whatever network you want to interact with
+  const sdk = new BadgerSDK({
+    network: Network.Ethereum,
+    provider:
+      'https://eth-archival.gateway.pokt.network/v1/lb/<APP_ID>',
+  });
+  await sdk.ready();
+
+  // load all badger productionized vaults - these appear on the app
+  const productionVaults = await sdk.registry.getProductionVaults();
```
[the remark this answers is not in the document]
Comment (NamelsKing) [reply examples/harvests.ts:17]: nit

[the remark this answers is not in the document]
Comment (axejintao) [reply examples/harvests.ts:17]: https://github.com/Badger-Finance/badger-sdk/issues/87

**Pull request merged**